va/va.go

```diff
@@ -342,6 +349,50 @@ func (va VAImpl) validateDNS01(task *vaTask) *core.ValidationRecord {
 	return result
 }
 
+func (va VAImpl) validateDNSAccount01(task *vaTask) *core.ValidationRecord {
+	acctHash := sha256.Sum256([]byte(task.AccountURL))
+	acctLabel := strings.ToLower(base32.StdEncoding.EncodeToString(acctHash[0:10]))
+	scope := "host"
+	if task.Wildcard {
+		scope = "wildcard"
+	}
+	challengeSubdomain := "_" + acctLabel +
+		"._acme-" + scope + "-challenge." + task.Identifier.Value
```

Comment:
We prefer using sprintf (see line 316):
```suggestion
	challengeSubdomain := fmt.Sprintf("_%s._acme-%s-challenge.%s", acctLabel, scope, task.Identifier.Value)
```